raiden/transfer/events.py

```diff
@@ -143,28 +143,28 @@ def __ne__(self, other):
 class ContractSendChannelBatchUnlock(ContractSendEvent):
     """ Event emitted when the lock must be claimed on-chain. """
 
-    def __init__(self, token_network_identifier, channel_identifier, merkle_treee_leaves):
+    def __init__(self, token_network_identifier, channel_identifier, merkle_tree_leaves):
         self.token_network_identifier = token_network_identifier
         self.channel_identifier = channel_identifier
-        self.merkle_treee_leaves = merkle_treee_leaves
+        self.merkle_tree_leaves = merkle_tree_leaves
```

Comment:
I believe this is a breaking change, could you make a new PR for this? (Or we can merge @palango 's PRs for this release too)